misc/python/materialize/parallel_workload/database.py

```diff
@@ -473,5 +677,16 @@ def create_relations(self, exe: Executor) -> None:
         for row in exe.cur.fetchall():
             exe.execute(f"DROP ROLE {row[0]}")
 
+        exe.execute("CREATE CONNECTION kafka_conn FOR KAFKA BROKER 'kafka:9092'")
+        exe.execute(
+            "CREATE CONNECTION csr_conn FOR CONFLUENT SCHEMA REGISTRY URL 'http://schema-registry:8081'"
```

Comment:
If we connect to Kafka from outside of the docker-compose context we have to use the forwarded port, so `f"localhost:{ports['kafka']}"`, but from inside the docker-compose context Kafka is available on the `kafka` hostname with standard `9092` port. Since environmentd is running inside of docker-compose, this is actually correct.

Reply:
Ok